Document limitations to OIDC session sync (#21753)
opf/openproject: docs/system-admin-guide/authentication/openid-providers/README.md

# docker-compose run --rm web bundle exec rails console
```

Once in the console you can then enter the following to enable the setting and leave the console.

```shell
Setting.oauth_allow_remapping_of_existing_users = true
exit
```

Then, existing users should be able to log in using their Azure identity. Note that this works only if the user is using password-based authentication, and is not linked to any other authentication source (e.g. LDAP) or OpenID provider.

Note that this setting is set to true by default for new installations already.

> **Q: How to automatically log out users from OpenProject that I deleted from the SSO provider?**

Reviewer: ```suggestion
**Q: How can I automatically log users out of OpenProject after I delete them from the SSO provider?**
```

> A: OpenProject does not currently revalidate user sessions after the initial login of a user. So even if the session at the SSO provider expires

Reviewer: ```suggestion
A: OpenProject does not currently revalidate user sessions after the initial login of a user. So even if the SSO provider session expires
```

> or the user is removed from there, this will not immediately have an effect in OpenProject. [A feature request exists](https://community.openproject.org/wp/65072) to improve this.

Reviewer: ```suggestion
or the user is removed from the SSO provider, this will not immediately have an effect in OpenProject. [This feature  was requested](https://community.openproject.org/wp/65072) to improve this flow.
```

Author: > This feature

I am wondering whether this suggestion makes sense when reading the sentence. "This feature" doesn't relate to anything outside the linking context, so you have to know that the "this" refers to the linked work package to understand the sentence.

Reviewer: We can also say "A feature was requested" instead.

Author: Okidoki. Will go for

```suggestion
or the user is removed from the SSO provider, this will not immediately have an effect in OpenProject. [A feature  was requested](https://community.openproject.org/wp/65072) to improve this flow.
```
